### PR TITLE
Add Python Airflow API token instructions

### PR DESCRIPTION
**Pull request by fritz-astronomer**

### CODE REVIEW

**Comment by fritz-astronomer** [comment]
oh - looking at the rest of the page, we should give the same treatment to the other code samples (that happen to have curl/requests already 😅 )

File: astro/airflow-api.mdx [hunk 1]
```diff
@@ -36,17 +39,38 @@ Calling the Airflow REST API for a Deployment requires:
 ### Retrieve an access token
 
 To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret:
-
+<Tabs>
+<TabItem value="sh" label="Shell">
+Using [cURL](https://curl.se/):
```
Comment (jwitz) [inline astro/airflow-api.mdx:44]:
```suggestion

<TabItem value="sh" label="Shell">

Using [cURL](https://curl.se/):

```
I get good luck with builds when I add space between all different kinds of elements.

**Review by jwitz — CHANGES_REQUESTED** [review]
@fritz-astronomer I'm trying to figure out the build error here.. Is there any reason you chose to replace the placeholder syntax with brackets?

Similarly, any reason we've switched to `mdx` here since we don't have to use partials?

File: astro/airflow-api.mdx [hunk 2]
```diff
@@ -36,17 +39,41 @@ Calling the Airflow REST API for a Deployment requires:
 ### Retrieve an access token
 
 To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret:
+<Tabs groupId="retrieve-an-access-token">
+
+<TabItem value="sh" label="Shell">
+
+Using [cURL](https://curl.se/):
```
Comment (jwitz) [inline astro/airflow-api.mdx:46]:
```suggestion
<Tabs groupId="retrieve-an-access-token">

<TabItem value="sh" label="Shell">

To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret using [cURL](https://curl.se/):

```

File: astro/airflow-api.mdx [hunk 3]
```diff
@@ -36,17 +39,41 @@ Calling the Airflow REST API for a Deployment requires:
...
-            "client_secret": "<api-key-secret>",
+            "client_id": "[api-key-id]",
+            "client_secret": "[api-key-secret]",
             "audience": "astronomer-ee",
             "grant_type": "client_credentials"}'
 ```
-
+</TabItem>
+<TabItem value="python" label="Python">
+Using the [`requests`](https://docs.python-requests.org/en/latest/index.html) library:
```
Comment (jwitz) [inline astro/airflow-api.mdx:59]:
```suggestion

To retrieve an Astro access token, use the [`requests`](https://docs.python-requests.org/en/latest/index.html) library to make your API request. For example, your code might look like the following:
```

File: astro/airflow-api.mdx [hunk 4]
```diff
@@ -36,17 +39,41 @@ Calling the Airflow REST API for a Deployment requires:
...
+```
```
Comment (jwitz) [inline astro/airflow-api.mdx:60]:
```suggestion

```python
```

File: astro/airflow-api.mdx [hunk 5]
```diff
@@ -36,17 +39,41 @@ Calling the Airflow REST API for a Deployment requires:
...
+          "audience": "astronomer-ee",
+          "grant_type": "client_credentials"
+      }
+  )
+  r.raise_for_status()
+  return r.json()
+```
+</TabItem>
+</Tabs>
+
```
Comment (jwitz) [inline astro/airflow-api.mdx:76]:
```suggestion

</TabItem>
</Tabs>
```

File: astro/airflow-api.md [hunk 6]
```diff
@@ -35,18 +38,44 @@ Calling the Airflow REST API for a Deployment requires:
...
+
+To retrieve an Astro access token, run the following API request with your Deployment API key ID and secret using [cURL](https://curl.se/):
+
 
 ```sh
 curl --location --request POST "https://auth.astronomer.io/oauth/token" \
         --header "content-type: application/json" \
         --data-raw '{
-            "client_id": "<api-key-id>",
+            "client_id": "<api-key-id]",
```
Comment (jwitz) [inline astro/airflow-api.md:52]:
```suggestion
            "client_id": "<api-key-id>",
```

File: astro/airflow-api.md [hunk 7]
```diff
@@ -185,7 +214,7 @@ print(response.json())
 
 You can pause a DAG by executing a `PATCH` command against the [`dag` endpoint](https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/patch_dag).
 
-Replace `<your-dag-id>` with your own value.
+Replace `[your-dag-id]` with your own value.
```
Comment (jwitz) [inline astro/airflow-api.md:217]:
```suggestion
Replace `<your-dag-id>` with your own value.
```

**Review by jwitz — APPROVED** [review]

**Pull request merged**